rpc/documentation/api.md

```diff
@@ -376,6 +384,45 @@ should begin at.
 - `FailedPrecondition`: The wallet or consensus RPC server has not been opened.
 
 **Stability:** Unstable
+___
+
+#### `FetchMissingCFilters`
+
+The `FetchMissingCFilters` method fetches any missing committed filters from
+the main chain.
+
+**Request:** `FetchMissingCFiltersRequest`
+
+**Response:** `FetchMissingCFiltersResponse`
+
+**Expected errors:**
+
+- `FailedPrecondition`: The wallet or consensus RPC server has not been opened.
+
+**Stability:** Unstable
+___
+
+#### `RescanPoint`
+
+The `RescanPoint` method returns the current RescanPoint hash which allows other
+methods to know where the wallet has last been rescanned through.  This allows
+clients to easily discern where startup rescan should be started from or
+whether DiscoverAddresses is required to run again.
```

Comment:
This second sentence could read "A non-null rescan point indicates that blocks currently in the main chain must be checked for address usage and relevant transactions."